### PR TITLE
[docs] improve descriptions for our APIs, and add videos

### PR DESCRIPTION
**Pull request by cruzach**
# Why

People should be able to navigate to an API and know instantly what the module is for, capable of, etc.

For some of the ones where I thought it would help, I added videos (ViewPager, DocumentPicker, ImagePicker, DateTimePicker, MailComposer, Sharing)

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/camera.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 
-A React component that renders a preview for the device's either front or back camera. Camera's parameters like zoom, auto focus, white balance and flash mode are adjustable. With use of `Camera` one can also take photos and record videos that are saved to the app's cache. Morever, the component is also capable of detecting faces and bar codes appearing on the preview. Check out a full example at [expo/camerja](https://github.com/expo/camerja). You can try it with Expo at [@documentation/camerja](https://expo.io/@documentation/camerja).
+**`expo-camera`** provides a React component that renders a preview for the device's front or back camera. The camera's parameters like zoom, auto focus, white balance and flash mode are adjustable. With the use of `Camera`, one can also take photos and record videos that are then saved to the app's cache. Morever, the component is also capable of detecting faces and bar codes appearing in the preview. Check out a full example at [expo/camerja](https://github.com/expo/camerja). You can try it with Expo at [@documentation/camerja](https://expo.io/@documentation/camerja).
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/camera.md]: it'd be good to move this example to a snack if possible

File: docs/pages/versions/unversioned/sdk/gl-view.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
 title: GLView
-sourceCodeUrl: "https://github.com/expo/expo/tree/sdk-36/packages/expo-gl"
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-gl'
 ---
 
 import TableOfContentSection from '~/components/plugins/TableOfContentSection';
 
-A `View` that acts as an OpenGL ES render target. On mounting, an OpenGL ES context is created. Its drawing buffer is presented as the contents of the `View` every frame.
+**`expo-gl`** provides a `View` that acts as an OpenGL ES render target, useful for rendering 2D and 3D vector graphics. On mounting, an OpenGL ES context is created. Its drawing buffer is presented as the contents of the `View` every frame.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/gl-view.md]:
```suggestion
**`expo-gl`** provides a `View` that acts as an OpenGL ES render target, useful for rendering 2D and 3D graphics. On mounting, an OpenGL ES context is created. Its drawing buffer is presented as the contents of the `View` every frame.
```

File: docs/pages/versions/unversioned/sdk/intent-launcher.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
 ---
 title: IntentLauncher
-sourceCodeUrl: "https://github.com/expo/expo/tree/sdk-36/packages/expo-intent-launcher"
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-intent-launcher'
 ---
 
 import TableOfContentSection from '~/components/plugins/TableOfContentSection';
 
-Provides a way to launch android intents. e.g. - opening a specific settings screen.
+**`expo-intent-launcher`** provides a way to launch android intents. For example- opening a specific settings screen.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/intent-launcher.md]:
```suggestion
**`expo-intent-launcher`** provides a way to launch Android intents. For example, you can use this API to open a specific settings screen.
```

File: docs/pages/versions/unversioned/sdk/mail-composer.md [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
+<Video file={"sdk/mailcomposer.mp4"} loop={false} />
 
 #### Platform Compatibility
 
-| Android Device | Android Emulator | iOS Device | iOS Simulator |  Web  |
-| ------ | ---------- | ------ | ------ | ------ |
-| ✅     |  ✅     | ✅     | ✅     | ✅    |
+| Android Device | Android Emulator | iOS Device | iOS Simulator | Web |
+| -------------- | ---------------- | ---------- | ------------- | --- |
+| ✅             | ✅               | ✅         | ✅            | ✅  |
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/mail-composer.md]: should we put a red x here for iOS simulator, given the description above?

File: docs/pages/versions/unversioned/sdk/pedometer.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
 ---
 title: Pedometer
-sourceCodeUrl: "https://github.com/expo/expo/tree/sdk-36/packages/expo-sensors"
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-sensors'
 ---
 
 import SnackEmbed from '~/components/plugins/SnackEmbed';
 
-Uses Core Motion on iOS and the system `hardware.Sensor` on Android to get the user's step count.
+`Pedomater` from **`expo-sensors`** uses Core Motion on iOS and the system `hardware.Sensor` on Android to get the user's step count, and also allows you to subscribe to pedometer updates.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/pedometer.md]:
```suggestion
`Pedometer` from **`expo-sensors`** uses Core Motion on iOS and the system `hardware.Sensor` on Android to get the user's step count, and also allows you to subscribe to pedometer updates.
```

File: docs/pages/versions/unversioned/sdk/splash-screen.md [hunk 6]
```diff
@@ -1,17 +1,17 @@
 ---
 title: SplashScreen
-sourceCodeUrl: "https://github.com/expo/expo/blob/sdk-36/packages/expo/src/launch/SplashScreen.ts"
+sourceCodeUrl: 'https://github.com/expo/expo/blob/sdk-36/packages/expo/src/launch/SplashScreen.ts'
 ---
 
-A module that tells Expo to keep the splash screen visible until you make it hide.
+The `SplashScreen` module tells Expo to keep the splash screen visible until you make it hide.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/splash-screen.md]:
```suggestion
The `SplashScreen` module tells Expo to keep the splash screen visible until you choose to hide it. This is useful to do some work behind the scenes before displaying your app and to create transitions for your splash screen, so you can have it fade out or slide away, for example.
```

File: docs/pages/versions/unversioned/sdk/task-manager.md [hunk 7]
```diff
@@ -1,19 +1,19 @@
 ---
 title: TaskManager
-sourceCodeUrl: "https://github.com/expo/expo/tree/sdk-36/packages/expo-task-manager"
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-task-manager'
 ---
 
-An API that allows to manage tasks, especially these running while your app is in the background.
-Some features of this module are used by other modules under the hood. Here is a list of modules using TaskManager:
+**`expo-task-manager`** provides an API that allows you to manage tasks, especially those running while your app is in the background.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/task-manager.md]:
```suggestion
**`expo-task-manager`** provides an API that allows you to manage long-running tasks, in particular those tasks that can run while your app is in the background.
```

File: docs/pages/versions/unversioned/sdk/video-thumbnails.md [hunk 8]
```diff
@@ -1,13 +1,15 @@
 ---
 title: VideoThumbnails
-sourceCodeUrl: "https://github.com/expo/expo/tree/sdk-36/packages/expo-video-thumbnails"
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-video-thumbnails'
 ---
 
+**`expo-video-thumbnails`** allows you to generate an image to serve as a thumbnail for a provided video.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/video-thumbnails.md]:
```suggestion
**`expo-video-thumbnails`** allows you to generate an image to serve as a thumbnail from a video file.
```

File: docs/pages/versions/unversioned/sdk/video.md [hunk 9]
```diff
@@ -1,17 +1,17 @@
 ---
 title: Video
-sourceCodeUrl: "https://github.com/expo/expo/tree/sdk-36/packages/expo-av"
+sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-av'
 ---
 
-A component that displays a video inline with the other React Native UI elements in your app. The display dimensions and position of the video on screen can be set using usual React Native styling.
+The `Video` component from **`expo-av`** displays a video inline with the other React Native UI elements in your app. The display dimensions and position of the video on screen can be set using usual React Native styling.
```
Comment (brentvatne) [inline docs/pages/versions/unversioned/sdk/video.md]:
```suggestion
The `Video` component from **`expo-av`** displays a video inline with the other UI elements in your app.
```

**Review by brentvatne — CHANGES_REQUESTED** [review]
👏

File: docs/pages/versions/unversioned/sdk/mail-composer.md (hunk 4 above)
Comment (cruzach) [reply docs/pages/versions/unversioned/sdk/mail-composer.md]: oops good catch

**Pull request merged**